src/store/index.js

```diff
@@ -14,6 +16,13 @@ const sortByVoteCount = (a, b) => {
   return 0;
 }
 
+// Firebase
+const firebaseConfig = {
+  databaseURL: "https://movie-log-8c4d5-default-rtdb.firebaseio.com",
```

Comment:
i might just be being paranoid but i'd store this url as an env variable if possible (i think surge does allow this, never done it myself)

Reply:
Well, this URL isn't meant to be secret. Anyone with a browser can monitor the network tab and get our address. 

The way to make the DB secure would be to set up some actual auth rules on Firebase. But that's kind of hard so I haven't done it.